content/community/gsoc-2022.en.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: XSF and Google Summer of Code 2022
+---
+
+[![](images/logos/GSoC_2022_Logo.png)](XSF and GSoC 2022 Logo)
+
+The XMPP Standards Foundation (XSF) is planning to participate as a hosting organisation in the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/).
+If you are interested to participate as student (or simply newcomer to open-source), mentor or XMPP project this is the page to start with!
+ 
+If you can imagine yourself part time (or even full time) working on an open-source software project during the summer, then consider to join an XMPP real-time communication technology project and participate in the Google Summer of Code 2022!
+
+### XMPP - the Extensible Messaging and Presence Protocol
+
+XMPP is the Extensible Messaging and Presence Protocol, a set of open technologies for instant messaging, presence, multi-party chat, voice and video calls, collaboration, lightweight middle-ware, content syndication, and generalized routing of XML data. XMPP was originally developed in the Jabber open-source community to provide an open, decentralized alternative to the closed instant messaging services at that time.
+
+### Google Summer of Code 2022
+
+The Google Summer of Code is an international annual program in which Google awards stipends to contributors who successfully complete a free and open-source software coding project during the summer. Project ideas are listed by host organizations involved in open-source software development, though students can also propose their own project ideas. 
+
+
+### First steps
+
+If you want to participate we recommoned the following preparations:
+
+- Read the resources at [Google's website](https://summerofcode.withgoogle.com/help) and review if this is something suitable to you and your plans this year.
+- Take a first look at our XMPP project detail page in the [XSF wiki](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022) (work in progress).
+- You can simply reach us [via web chat](https://xmpp.org/chat#converse/room?jid=gsoc@muc.xmpp.org) or review your ["Getting started with XMPP"](https://xmpp.org/getting-started/) page and reach us via our [public chat](xmpp:gsoc@muc.xmpp.org?join gsoc@muc.xmpp.org).
+
+Checkout our media channels!
+
+- [Twitter](https://twitter.com/xmpp)
+- [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
+- [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
+
+Looking forward and stay tuned!
+ –The XSF Organisation Admin
+
+### Sources
+
+- Google Summer of Code, Wikipedia contributors, Wikipedia, The Free Encyclopedia, 22 November 2021 23:03 UTC, 22 January 2022 14:31 UTC, https://en.wikipedia.org/w/index.php?title=Google_Summer_of_Code&oldid=1056637774, revision history statistics, 1056637774
```

Comment:
I would linkify the sources to get rid of the bare URLs in the rendered page.

Reply:
yep, way to detailed I think

Reply:
@vanitasvitae but do you think its legal to just link the permalink?